conan/api/subapi/config.py

```diff
@@ -208,7 +208,7 @@ def clean(self):
         packages_folder = self.global_conf.get("core.cache:storage_path") or os.path.join(self.home(), "p")
         for content in contents:
             content_path = os.path.join(self.home(), content)
-            if content_path == packages_folder or content == "version.txt":
+            if content_path == packages_folder:
```

Comment:
This makes it so we regenerate the folder structure of a clean Conan installation, which I think was a bug in the initial implementation which I found while debugging the migration